com.unity.render-pipelines.core/Editor/FilterWindow.cs

```diff
@@ -239,7 +239,7 @@ internal static bool ValidateAddComponentMenuItem()
             return true;
         }
 
-        internal static bool Show(Vector2 position, IProvider provider)
+        public static bool Show(Vector2 position, IProvider provider)
```

Comment:
Mising documentation API?

Reply:
hmm yeah, good point. I'll write up a short doc for this one.